jck/compiler.api/playlist.xml

```diff
@@ -119,4 +119,119 @@
 			<group>jck</group>
 		</groups>
 	</test>
+	<test>
+		<testCaseName>jck-compiler-api-java_rmi_multijvm</testCaseName>
+		<disables>
+			<disable>
+				<comment>Disabled on all platforms and Java levels due to backlog/issues/559. To be run manually by CR team</comment>
+				<impl>ibm</impl>
+			</disable>
+			<disable>
+				<comment>Disabled on all platforms and Java levels due to backlog/issues/559. To be run manually by CR team</comment>
+				<impl>openj9</impl>
+			</disable>
+		</disables>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>$(GEN_JTB) tests=api/java_rmi testsuite=COMPILER'; \
```

Comment:
There is `'` at the end. Is it intended?

Reply:
It is. The corresponding quote can be found in `GEN_JTB` definition under jck.mk.